src/site/antora/modules/ROOT/pages/jakarta.adoc

```diff
@@ -267,6 +267,14 @@ include::example$manual/webapp/AsyncServlet.java[tag=manual]
 [#log4j-taglib]
 === Logging in JavaServer Pages
 
+[WARNING]
+====
+*The Apache Log4j Tag library is planned to be removed in the next major release due to lack of interest*
+
+If you are using this library, consider forking the project or stepping up as maintainer.
+See xref:development.adoc[] for details.
```

Comment:
```suggestion
*The Log4j Tag library is planned to be removed in the next major release!*
If you are using this library, please get in touch with the Log4j maintainers using link:{logging-services-url}/support.html[the official support channels].
```

1. Let's not explicitly share a motivation, since in reality there are many.
2. I think it'd be better to direct users to us first, instead of directly telling them to fork it. This way we can get an idea about the users, if there are any.